gp-cli/src/main/java/com/ibm/g11n/pipeline/tools/cli/UpdateBundleCmd.java

```diff
@@ -34,34 +35,67 @@
 
     @Parameter(
             names = {"-l", "--languages"},
-            description = "List of bundle's target language ID(s) separted by comma.")
+            description = "List of bundle's target language ID(s) separted by comma. "
+                        + "Empty list \"\" will remove all existing target languages.")
     private String languageIdsListStr;
 
     @Parameter(
             names = {"-n", "--note"},
-            description = "Translation instruction note")
+            description = "Translation instruction note. "
+                        + "Empty note \"\" will remove existing instruction note.")
     private String note;
 
+    @Parameter(
+            names = {"-r", "--readOnly"},
+            description = "true to set the bundle read only.")
```

Comment:
Oh.. right. I changed the description several times - earlier version had both true/false, but I thought it's too redundant.. But, I agree that we should spell out valid value, because we only accept true or false here. I'll update this later.